### PR TITLE
Add padding around search result breadcrumbs

### PR DESCRIPTION
**Pull request by thesunshade**
This is my very first attempt at making a change on my local build of the site and pushing it to the main repository. Please let me know if I have done anything wrong.

Feel free to delete if need be.

### CODE REVIEW

**Comment by thesunshade** [comment]
This addresses https://github.com/suttacentral/suttacentral/issues/2982

File: client/elements/styles/sc-page-search-styles.js [hunk 1]
```diff
@@ -474,6 +474,8 @@ export const SCPageSearchStyles = css`
     transition: var(--sc-link-transition);
     opacity: 0.8;
     border-radius: 16px;
+    padding: 0 4px 0 4px;
```
[the remark this answers is not in the document]
Comment (khemarato) [reply client/elements/styles/sc-page-search-styles.js]: I'll agree with this.

**Comment by khemarato** [comment]
I'll also agree with the Robot's suggestion that it's best to include before and after screenshots for UI changes like this. Though, that's maybe not necessary for such a small diff as this, it does help get everyone on the same page with what is being changed here and it proves that you tested your change locally, which is confidence inspiring.

**Comment by thesunshade** [comment]
Huh. I think it worked?

So, I made this change on my local machine. This is what it looks like now:
image

This is what it looked like before:
image

**Comment by thesunshade** [comment]
Not sure what the failed tests are ¯\\\_(ツ)_/¯

**Comment by khemarato** [comment]
> Not sure what the failed tests are ¯\_(ツ)_/¯

Me neither. Will have to await @ihongda 's wisdom on that one 😊

**Comment by ihongda** [comment]
Congratulations on your first PR. @thesunshade 
Test passed!

**Pull request merged**